Benchmarks in ethcore-util don't compile

I tried running `cargo bench` inside `util` but it failed:

```rust
error[E0432]: unresolved import `ethcore_util::rlp::*`
  --> benches/rlp.rs:30:5
   |
30 | use ethcore_util::rlp::*;
   |     ^^^^^^^^^^^^^^^^^^^^^ Could not find `rlp` in `ethcore_util`

error[E0433]: failed to resolve. Use of undeclared type or module `RlpStream`
  --> benches/rlp.rs:37:20
   |
37 |            let mut stream = RlpStream::new();
   |                             ^^^^^^^^^^^^^^ Use of undeclared type or module `RlpStream`

error[E0433]: failed to resolve. Use of undeclared type or module `Rlp`
  --> benches/rlp.rs:48:13
   |
48 |            let rlp = Rlp::new(&data);
   |                      ^^^^^^^^ Use of undeclared type or module `Rlp`

error[E0433]: failed to resolve. Use of undeclared type or module `RlpStream`
  --> benches/rlp.rs:57:20
   |
57 |            let mut stream = RlpStream::new();
   |                             ^^^^^^^^^^^^^^ Use of undeclared type or module `RlpStream`

error[E0433]: failed to resolve. Use of undeclared type or module `Rlp`
  --> benches/rlp.rs:72:13
   |
72 |            let rlp = Rlp::new(&data);
   |                      ^^^^^^^^ Use of undeclared type or module `Rlp`

error[E0433]: failed to resolve. Use of undeclared type or module `RlpStream`
  --> benches/rlp.rs:81:20
   |
81 |            let mut stream = RlpStream::new_list(3);
   |                             ^^^^^^^^^^^^^^^^^^^ Use of undeclared type or module `RlpStream`

error[E0433]: failed to resolve. Use of undeclared type or module `Rlp`
  --> benches/rlp.rs:94:13
   |
94 |            let rlp = Rlp::new(&data);
   |                      ^^^^^^^^ Use of undeclared type or module `Rlp`

error[E0433]: failed to resolve. Use of undeclared type or module `RlpStream`
   --> benches/rlp.rs:106:20
    |
106 |           let mut stream = RlpStream::new_list(1000);
    |                            ^^^^^^^^^^^^^^^^^^^ Use of undeclared type or module `RlpStream`

error: aborting due to 8 previous errors

Build failed, waiting for other jobs to finish...
error: Could not compile `ethcore-util`.
````

What am I doing wrong?

--- a/util/benches/bigint.rs
+++ b/util/benches/bigint.rs
@@ -24,7 +24,7 @@
 #![feature(asm)]
 
 extern crate test;
-extern crate bigint;
+extern crate ethcore_bigint as bigint;
 
 use test::{Bencher, black_box};
 use bigint::uint::{U256, U512, Uint, U128};

--- a/util/benches/rlp.rs
+++ b/util/benches/rlp.rs
@@ -23,12 +23,13 @@
 #![feature(test)]
 
 extern crate test;
-extern crate ethcore_util;
+extern crate rlp;
+extern crate ethcore_bigint as bigint;
 
 use test::Bencher;
 use std::str::FromStr;
-use ethcore_util::rlp::*;
-use ethcore_util::U256;
+use rlp::*;
+use bigint::uint::U256;
 
 #[bench]
 fn bench_stream_u64_value(b: &mut Bencher) {

